feat(nestjs-json-rpc): default transport for rpc

klerick opened the pull request:
- http transport
- validation
- explorer need service

Closes: #77

klerick committed:
feat(nestjs-json-rpc): default transport for rpc

- http transport
- validation
- explorer need service

Closes: #77

klerick committed:
feat(nestjs-json-rpc-sdk): sdk for rpc

- http transport
- factory for native js
- angular module

Closes: #77

klerick committed:
ci(nestjs-json-rpc-sdk): sync package-lock.json

klerick committed:
ci(nestjs-json-rpc-sdk): fix test and create e2e

klerick committed:
ci(nestjs-json-rpc-sdk): exclude type-for-rpc from test run

klerick committed:
feat(nestjs-json-rpc,nestjs-json-rpc-sdk): add ws transport

Add WebSocket transport for backend
Add Websocket transport for sdk,

klerick committed:
feat(nestjs-json-rpc-sdk): add takeUntil for disconnect socket

klerick committed:
test: Fix tests

klerick committed:
docs(nestjs-json-rpc,nestjs-json-rpc-sdk): Add readme

Dzhara commented on apps/json-api-server/src/app/rpc/service/rpc.service.ts:
```diff
@@ -0,0 +1,31 @@
+
+@RpcHandler()
+export class RpcService implements IRpcService {
+  methodeWithObjectParams(a: InputType): Promise<OutputType> {
```
typo in word "method", in some other places as well. can do word search by "methode" and correct to "method"

klerick committed:
docs(nestjs-json-rpc,nestjs-json-rpc-sdk): Add readme

klerick committed:
ci: try fix nx cache

klerick committed:
fix(json-api-nestjs): Fix validation

Fix validation if try create conditional for date filed. If check field for null validation throw error: "String should be as date"

Closes: #81

actions-user committed:
chore(release): publish

- project: json-api-nestjs 7.0.1

klerick committed:
ci: remove command ls from ci

klerick committed:
ci(nestjs-json-rpc-sdk): Prepare task for publish npm

klerick committed:
chore(nestjs-json-rpc,nestjs-json-rpc-sdk): Prepare npm package before publish
